src/helpers/browser.js

```diff
@@ -90,6 +91,26 @@ export function isMobileBrowser() {
   return browsers.mobile.value;
 }
 
+/**
+ * @returns {boolean}
+ */
+export function isIOS() {
+  return platforms.ios.value;
+}
+
+/**
+ * A hacky way to recognize the iPad. Since iOS 13, the iPad on Safari mimics macOS behavior and user agent.
+ *
+ * @see {@https://stackoverflow.com/a/57838385}
+ * @param {object} [metaObject] The browser identity collection.
+ * @returns {boolean}
+ */
+export function isIpadOS({ maxTouchPoints } = navigator) {
+  return maxTouchPoints &&
+  maxTouchPoints > 2 &&
+  platforms.mac.value;
```

Comment:
Correct me if I'm wrong but this can be simplified to this?
```suggestion
  return maxTouchPoints > 2 && platforms.mac.value;
```

Reply:
You're right, I think that's safe enough.